src/Time/Actions/RecordTimeStepperData.hpp

```diff
@@ -70,7 +70,8 @@ struct RecordTimeStepperData {
            const gsl::not_null<db::item_type<history_tag>*> history,
            const db::const_item_type<variables_tag>& vars,
            const db::const_item_type<Tags::TimeStepId>& time_step_id) noexcept {
-          history->insert(time_step_id, vars, std::move(*dt_vars));
+            auto dt_vars_copy = *dt_vars;
```

Comment:
@moxcodes you will need to rebase as I merged #2137 .  If this commit survives, it should have a better commit message...